cli/internal/globby/globby_test.go

```diff
@@ -396,12 +397,8 @@ func TestGlobFilesFs(t *testing.T) {
 				includePatterns: []string{"../spanish-inquisition/**", "dist/**"},
 				excludePatterns: []string{},
 			},
-			want: []string{
-				"/repos/some-app/dist/index.html",
-				"/repos/some-app/dist/js/index.js",
-				"/repos/some-app/dist/js/lib.js",
-				"/repos/some-app/dist/js/node_modules/browserify.js",
-			},
+			want:    []string{},
+			wantErr: true,
```

Comment:
These represent behavior changes. Previously we would silently proceed and now we fail.

Likely easy fixes for anybody who encounters it, but possibly noisy/disruptive on upgrade.